python/paddle/fluid/distributed/launch.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
can delete same file under tools

Reply:
better to make usage like: 

`python -m fluid.distributed.launch train.py --your args`

Reply:
done.